man/docker-daemon.8.md

```diff
@@ -0,0 +1,402 @@
+% DOCKER(1) Docker User Manuals
```

Comment:
Should this be `DOCKER DAEMON (8)`, or something along that line (not too well known with man conventions)